modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/registry/ObjectServiceUpgradeStepRegistrator.java

```diff
@@ -645,6 +645,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			"10.21.0", "10.22.0",
 			UpgradeProcessFactory.addColumns(
 				"ObjectDefinition", "enableObjectEntrySubscription BOOLEAN"));
+
+		registry.register(
+			"10.22.0", "10.22.1",
+			new com.liferay.object.internal.upgrade.v10_22_1.
+				ResourceActionUpgradeProcess());
```

Comment:
I believe it is a minor change instead of micro. See this [article](https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Schema-version).